pygfx/objects/_base.py

```diff
@@ -521,82 +521,133 @@ def _get_bounds_from_geometry(self):
             self._bounds_geometry = None
         return self._bounds_geometry
 
-    def get_geometry_bounding_box(self) -> np.ndarray | None:
+    def get_bounds_local(self):
+        """Get the Bounds based on this objects's geometry only, or None."""
+        return self._get_bounds_from_geometry()
+
+    def get_bounds_local_recursive(self):
+        """Get the Bounds, local to this objects space, of itself and all its children."""
+        # TODO: this replicates get_bounding_box(), but should it not include this local.matrix??
```

Comment:
All bounding geometry should be local - in the space of the geometry buffers.  It's much cheaper to transform a ray into this space than to transform bounding geometry into ray space (may or may not be world).